doc/services/device_mgmt/smp_groups/smp_group_1.rst

```diff
@@ -295,7 +295,7 @@ where:
     +-----------+--------------------------------------------------------------------------------+
     | "len"     | optional length of an image. Must appear when "off" is 0.                      |
     +-----------+--------------------------------------------------------------------------------+
-    | "off"     | offset of image chunk the request carries.                                     |
+    | "off"     | offset of the next expected chunk of application image.                        |
```

Comment:
I have no idea what's going on here but no

Reply:
Yeah I was trying to edit from GitHub and found the wrong line or something.  Will fix from an editor ASAP.